setuptools/tests/test_virtualenv.py

```diff
@@ -69,7 +69,9 @@ def _get_pip_versions():
         'pip==9.0.3',
         'pip==10.0.1',
         'pip==18.1',
-        'pip==19.0.1',
+        # fails due to pypa/pip#6599
+        # 'pip==19.3.1',
```

Comment:
Maybe xfail it instead?
```suggestion
        pytest.mark.xfail('pip==19.3.1', reason='fails due to pypa/pip#6599'),
```